app/schemas/models/campaign.schema.js

```diff
@@ -122,7 +122,19 @@ _.extend(CampaignSchema.properties, {
       }
 
     }
-  }
+  },
+  scenarios: {
+    type: 'object',
```

Comment:
let's use array for a list instead of doing Object.values etc

Reply:
scenarios format copied from `campaign.levels`